helm/default-apps-gcp/values.yaml

```
@@ -113,7 +113,7 @@ apps:
     namespace: kube-system
     # used by renovate
     # repo: giantswarm/kube-state-metrics-app
```

```suggestion
    # repo: giantswarm/kube-state-metrics-app
```

We are not relying on `renovate` automation to upgrade these apps. I'm not 100% sure how it works, but I think the comment is relevant. Can you try changing the comment to use the right app name? 
I see that the repository is called `kube-state-metrics-app`, would this a problem @erkanerol ?

---

The renovate bot only updates versions. As far as I understand, they renamed the chart name recently here:
https://github.com/giantswarm/kube-state-metrics-app/pull/150/files#diff-3df0b31e07274c4b8007edbe5d694a00d7bf2d73a16f3e64c4e9039633d5a369

---

so what should the comment be?

```
# repo: giantswarm/kube-state-metrics
```

or

```
# repo: giantswarm/kube-state-metrics-app
```

---

It refers repo name so it should be `giantswarm/kube-state-metrics-app`